docs/contributing/CONTRIBUTING.md

```diff
@@ -1,42 +1,38 @@
-## Welcome to Invoke AI
-
-We're thrilled to have you here and we're excited for you to contribute. 
+# How to Contribute
 
+## Welcome to Invoke AI
 Invoke AI originated as a project built by the community, and that vision carries forward today as we aim to build the best pro-grade tools available. We work together to incorporate the latest in AI/ML research, making these tools available in over 20 languages to artists and creatives around the world as part of our fully permissive OSS project designed for individual users to self-host and use.
 
-Here are some guidelines to help you get started:
-
-### Technical Prerequisites
 
-Front-end: You'll need a working knowledge of React and TypeScript.
+## Contributing to Invoke AI
+Anyone who wishes to contribute to InvokeAI, whether features, bug fixes, code cleanup, testing, code reviews, documentation or translation is very much encouraged to do so.
 
-Back-end: Depending on the scope of your contribution, you may need to know SQLite, FastAPI, Python, and Socketio. Also, a good majority of the backend logic involved in processing images is built in a modular way using a concept called "Nodes", which are isolated functions that carry out individual, discrete operations. This design allows for easy contributions of novel pipelines and capabilities.
+To join, just raise your hand on the InvokeAI Discord server (#dev-chat) or the GitHub discussion board.
```

Comment:
Does every new user have posting rights to `dev-chat` immediately? Coz if they don't, this might need a bit of tweaking. @hipsterusername

Reply:
I believe it was updated to give all users access

Reply:
Oh, didn't know that. If it did, then all good.